scripts/process/find_range.R

```diff
@@ -0,0 +1,18 @@
+#' Extract the min and max to use to scale circles in D3
+#' 
+process.find_range <- function(viz) {
+  
+  deps <- readDepends(viz)
+  wu_data_15_simple <- deps[["wu_data_15_simple"]]
+  col_names <- deps[["col_names"]][["new_names"]]
+  
+  water_use_df <- dplyr::select(wu_data_15_simple, col_names)
+  # wu_data_15_range <- data.frame(
+  #   min = min(water_use_df, na.rm = TRUE),
+  #   max = max(water_use_df, na.rm = TRUE),
+  #   stringsAsFactors = FALSE
+  # )
```

Comment:
looks like these lines could be removed now?

Reply:
oops - yes. I was testing different formats